common/containers/TabSection/Notifications.tsx

```diff
@@ -5,28 +5,38 @@ import {
 } from 'actions/notifications';
 import React from 'react';
 import { connect } from 'react-redux';
+import { TransitionGroup, CSSTransition } from 'react-transition-group';
 import NotificationRow from './NotificationRow';
 import './Notifications.scss';
 
 interface Props {
   notifications: Notification[];
   closeNotification: TCloseNotification;
 }
+
+const Transition = props => (
+  <CSSTransition
+    {...props}
+    classNames="animation"
```

Comment:
I would pick a more descriptive class than `animation`, this is likely to collide with some other usage of this elsewhere. Maybe `Notification` or something specific like that?